### PR TITLE
Tweak up for Workiva

### PR DESCRIPTION
**Pull request by robbecker-wf**
# Summary

Adds the required Workiva files for CI
- Dockerfile for Workiva Build
- skynet.yaml for Skynet and release pipeline
- aviary.yaml for Infosec

# Updates to allow analyzer 1
- Updated the http_parser version range to allow v4
- removed the dependency on `test_coverage` which is old and a flutter specific package

# Updates to allow Workiva Build to pass (You can test the commands locally)
- skipped a test that was failing, the rest pass

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+WORKDIR /build/dio
+RUN timeout 5m dart pub get
+RUN dart test -p vm
+RUN dart analyze
+RUN create_publishable_artifact.sh
+
+ARG BUILD_ARTIFACTS_PUB=/build/dio/pub_package.pub.tgz
+
+FROM scratch
```
Comment (robbecker-wf) [inline Dockerfile:32]: Basic Dockerfile to pub get, analyze, run tests, create an artifact.

**Comment by semveraudit-wf** [comment]
### Public API Changes

:scream: Error

Semver Audit Service experienced an error during its report.

For details, view the service logs for the correlation ID: [40150af0-e7ff-46a5-9cf7-2549c2e63203](https://splunk.workiva.net/en-US/app/search/search?q=search%20index%3D%22kinesis-wk-dev%22%20%20%22service.name%22%3Dsemver-audit%20%22context.correlationId%22%3D40150af0-e7ff-46a5-9cf7-2549c2e63203)



Showing results for c7d63d9d22704ac9afa9f6e6fc38eeea8b30bcd8
> Powered by [semver-audit-service](https://github.com/Workiva/semver-audit-service). Please report any problems by [filing an issue](https://github.com/Workiva/semver-audit-service/issues).
> Reported by the dart semver audit client 2.2.2
[Browse public API](https://semver-audit.wk-dev.wdesk.org/Workiva/w_dio/c7d63d9d22704ac9afa9f6e6fc38eeea8b30bcd8).

_Last edited UTC Jan 06 at 19:57:45_

**Review by todbachman-wf — APPROVED** [review]

**Comment by robbecker-wf** [comment]
QA+1 Workiva CI is now running and passing!

**Comment by robbecker-wf** [comment]
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**